page-components/links.md

```diff
@@ -158,18 +158,33 @@ Use the external link minicon (glyph E610) to emphasize a non-CFPB webpage.
 
 External links should open in the same browser window (i.e., do not set them to ```target="_blank"```). This allows the user to choose whether they want to open an additional window in order to view the content.
 </div>
+
 <div class="content-50 content-last regular-ex">
-<a class="link-with-icon" href="#">External link <span class="cf-icon cf-icon-external-link"></span></a>
+  <a class="a-link
+            a-link__icon
+            cf-icon
+            cf-icon__after
+            cf-icon-external-link" href="#">
+    <span class="a-link_text">External link</span>
+  </a>
 </div>
 
 ---
 
 <div class="content-50 content-first">
 ##### Email links
+
 Use the Email link minicon (glyph E302) to emphasize a <code>mailto</code> link.
 </div>
+
 <div class="content-50 content-last regular-ex">
-<a class="link-with-icon" href="#">Email us <span class="cf-icon cf-icon-email"></span></a>
+  <a class="a-link
+            a-link__icon
+            cf-icon
+            cf-icon__after
+            cf-icon-email" href="#">
+    <span class="a-link_text">Email us </span>
```

Comment:
Left an extra space at the end of the `span` here that's causing the underline to extend between the text and the icon.